qiskit_cold_atom/synqs_fermions/fqs_gate_library.py

```diff
@@ -0,0 +1,82 @@
+from typing import Union, Optional, List
+
+from qiskit.circuit.gate import Gate
+from qiskit_cold_atom import QiskitColdAtomError, add_gate
+import numpy as np
+
+class LoadGate(Gate):
+    """The load gate."""
+
+    def __init__(self) -> None:
+        """Create a new gate.
+
+        Args:
+            params: A list of parameters.
+        """
+
+        super().__init__(name="load", num_qubits=1, params=[], label=None)
+
+@add_gate
+def load(self, wire):
+    # pylint: disable=invalid-name
+    """add the load gate to a QuantumCircuit"""
+    return self.append(LoadGate(), [wire], [])
+
+class InterGate(Gate):
+    """The inter gate."""
+
+    def __init__(self,wires:List[int], theta_u:float) -> None:
+        """Create a new gate.
+
+        Args:
+            params: A list of parameters.
+        """
+        assert len(wires) == 8, 'InterGate must act on all modes which is 8.'
+
+        super().__init__(name="int", num_qubits=len(wires), params=[theta_u], label=None)
+
+@add_gate
+def inter(self, wires:List[int], theta_u:float):
+    # pylint: disable=invalid-name
+    """add the inter gate to a QuantumCircuit"""
+    return self.append(InterGate(wires,theta_u), wires)
+
+class HopGate(Gate):
```

Comment:
How does this relate to the existing fermion gates https://github.com/Qiskit-Extensions/qiskit-cold-atom/blob/410332a470ad371eab0cb74c2b88ee39aae1b887/qiskit_cold_atom/fermions/fermion_gate_library.py#L342 ? We should try and use those as much as possible.